proxy/Makefile.am

```diff
@@ -40,6 +40,7 @@ AM_CPPFLAGS += \
   -I$(abs_srcdir)/hdrs \
   -I$(abs_srcdir)/shared \
   -I$(abs_top_srcdir)/mgmt \
+  -I$(abs_top_srcdir)/mgmt/rpc \
```

Comment:
Don't do this - use `#include <rpc/...>` instead. We're trying to reduce the massive list of include directories.

Reply:
updated.